src/main/java/me/ryanhamshire/GriefPrevention/EntityEventHandler.java

```diff
@@ -988,6 +968,22 @@ else if(attackerData.pvpImmune)
                                 if(sendErrorMessagesToPlayers) GriefPrevention.sendMessage(attacker, TextMode.Err, Messages.CantFightWhileImmune);
                                 return;
                             }
+                            // disallow players attacking tamed wolves (dogs) unless under attack by said wolf
+                            else if (tameable instanceof Wolf) {
```

Comment:
`getType` would be better here (faster than `instanceof` check)

Reply:
Will do